sdk/servicebus/service-bus/src/serviceBusClient.ts

```diff
@@ -79,37 +79,59 @@ export class ServiceBusClient {
   }
 
   /**
-   * Creates a receiver for an Azure Service Bus queue.
+   * Creates a receiver for an Azure Service Bus queue in peekLock mode.
+   *
+   * In peekLock mode, the receiver has a lock on the message for the duration specified on the
+   * queue.
```

Comment:
Can we say that the lock is also lost once the message is settled?

```suggestion
   * queue. The lock expires if the message is settled within this duration. An attempt to settle the message after
   * lock expiry results in an error.
```

Reply:
I modified this slightly:

```typescript
/*
   * In peekLock mode, the receiver has a lock on the message for the duration specified on the
   * queue.
   *
   * Messages that are not settled within the lock duration will be redelivered as many times as
   * the max delivery count set on the queue, after which they get sent to a separate dead letter
   * queue.
   *
   * You can settle a message by calling complete(), abandon(), defer() or deadletter() methods on
   * the message.
*/
```

And for dead letter entities:

```typescript
  /*
   * In peekLock mode, the receiver has a lock on the message for the duration specified on the
   * queue.
   *
   * In peekLock mode, the receiver has a lock on the message for the duration specified on the
   * queue. Messages that are not settled within the lock duration will be redelivered.
   *
   * You can settle a message by calling complete(), abandon() or defer() methods on
   * the message.
   */
```